inject/inject.vala

```diff
@@ -225,6 +225,41 @@ namespace Frida.Inject {
 
 			exit_code = 0;
 
+			var fd = stdin.fileno();
+#if WINDOWS
+			var inchan = new IOChannel.win32_new_fd (fd);
+#else
+			var inchan = new IOChannel.unix_new (fd);
+#endif
+			inchan.add_watch (IOCondition.IN, (source, condition) => {
+				string str_return = null;
+				size_t length = -1;
+				size_t terminator_pos = -1;
```

Comment:
These could be moved inside the `try` as they're unused outside it. (Actually I see now that they're nullable at the C API level, so would be better to pass `null`, assuming the vapi is correct.)